web/help/GuideUnit1.md

```diff
@@ -716,60 +746,65 @@ Here is a table of colors you can use:
     ~~~~~
 
     If you do this, remember that **`&`** means "in front of", so
-    the shape in front should come first, and your backdrop should
+    the shape in front should come first, while your backdrop should
     come last.
 
 Translation: Moving
 -------------------
 
 The second transformation you can use in your programs is translation.  The
 `translated` function shifts a picture up, down, left, or right.  Its inputs
-are the preimage, and two distances to move the picture.  The result of
-`translated` is a new picture with the same content as the preimage, but
-shifted either horizontally (that is, left or right) or vertically (that is,
-up or down) or both.
+are the preimage and two numbers, which are distances that we want
+to move the picture.  The result of `translated` transformation is a new picture 
+with the same content as the preimage, but shifted either horizontally 
+(left or right) or vertically (up or down) or both.
 
 When you give distances to the `translated` function, you will list the
-horizontal change first, and the vertical change second.  A useful tool for
-finding these numbers is the **coordinate plane**.
+horizontal change first, and the vertical change second.  
+
+A useful tool for finding these numbers is the **coordinate plane**.
 
 ![](coordinate-plane.png width=50%)
 
 !!! Tip: Get your own coordinate plane
     The coordinate plane is a picture that your computer knows about in
-    CodeWorld.  To make your own coordinate plane, use this code:
+    CodeWorld.  
+    
+To make your own coordinate plane, use this code
 
     ~~~~~ . clickable
     program = drawingOf(coordinatePlane)
     ~~~~~
 
-This tool combines of two number lines.
+This tool is a combination of two number lines:
+
+1. The first number line is *horizontal*, like the *x-axis*, with positive numbers 
```

Comment:
Saying "like the x axis" is weird, since it is literally the x axis.  What about this?

```
This tool is a combination of two number lines:

1. The first number line is called the *x* *axis*.  It is *horizontal*, with positive numbers
   to the right and negative numbers to the left.
2. The second number line is called the *y* *axis*.  It is *vertical*, with positive numbers
   on the top and negative numbers on the bottom.
```